src/pkg/runner/runner.go

```diff
@@ -170,7 +172,7 @@ func (r *Runner) importTasks(includes []map[string]string, dir string, setVariab
 		for _, taskToAdd := range tasksFile.Tasks {
 			for _, currentTasks := range r.TasksFile.Tasks {
 				if taskToAdd.Name == currentTasks.Name {
-					return fmt.Errorf("task loop detected")
+					return fmt.Errorf("task loop detected, ensure no cyclic loops in tasks or includes files")
```

Comment:
could always add `"%q in %q", taskToAdd.Name, includePath` (or some variation) to show end users _which_ task has a cyclical loop

Reply:
lol yeah, so thats what I had initially, but I think it would potentially make things more confusing, because it could be a task that you aren't even using, but since we process all of the includes and tasks, its hitting an infinite loop due to an includes that another task is referencing.

Reply:
Is it fair to say that the detected task loop has to do with `currentTasks.Name` though? If that's the case I think it makes sense to give the user some clue about where the loop is located (ie. give them a task name).

Reply:
discussed IRL don't love it, but good enough for now considering our includes and task loop detection implementation